mods/gen4/formats-data.js

```diff
@@ -838,7 +838,7 @@ exports.BattleFormatsData = {
 		tier: "NFE",
 	},
 	ampharos: {
-		randomBattleMoves: ["substitute", "hiddenpowerice", "focuspunch", "thunderbolt", "focusblast", "signalbeam"],
+		randomBattleMoves: ["substitute", "hiddenpowerice", "discharge", "thunderbolt", "focusblast", "signalbeam"],
```

Comment:
Yeah, Sub + 3 attacks is still good. Focus Punch is just UU tech for Chansey/Clefable though.